apps/src/javalab/Theater.js

```diff
@@ -34,6 +45,7 @@ export default class Theater {
       }
       case TheaterSignalType.GET_IMAGE: {
         // Open the photo prompter
+        this.prompterUploadUrl = data.detail.uploadUrl;
```

Comment:
instead of setting a field, could we just pass the `prompterUploadUrl` to `onPhotoPromptedFileSelected`?

Reply:
So `onPhotoPrompterFileSelected` is called externally by Javalab (from JavalabConsole, when file is selected) but only Theater knows the upload URL which is why I had it set on a field.